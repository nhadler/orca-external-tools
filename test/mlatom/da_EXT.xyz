16

C             0.4813024374103          -1.4349227612885           0.5137386536117
C             0.4859702737149           1.4344889056846           0.5129530071391
C             1.2680466082266          -0.7096089630187          -0.3161249847856
C             1.2698482503147           0.7063075171378          -0.3168590194909
C            -1.5743749532249          -0.6736952829099          -0.2267301172749
C            -1.5741402429035           0.6770058850694          -0.2253736447211
H             0.4107820982713          -2.5048539561628           0.4024004256024
H             0.1545008650554          -1.0465393829088           1.4604758694958
H             0.4182444345553           2.5045006285090           0.4008820574544
H             0.1575374270938           1.0474822961018           1.4596556145266
H             1.7590868456084          -1.2036512202944          -1.1442175065287
H             1.7624326531584           1.1982575357438          -1.1452660986280
H            -1.9999454494089          -1.2291294376367           0.5911010005786
H            -1.4029477841077          -1.2229351940318          -1.1358103575475
H            -1.9978808718237           1.2312093010793           0.5942018379777
H            -1.4017244596312           1.2282076255495          -1.1330462098024
