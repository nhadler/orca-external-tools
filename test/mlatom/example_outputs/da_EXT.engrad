#
# Number of atoms
#
16
#
# Total energy [Eh]
#
-2.345413496349e+02
#
# Gradient [Eh/Bohr] A1X, A1Y, A1Z, A2X, ...
#
-3.425496990623e-03
 7.641738358263e-03
-2.967134862090e-03
-3.332725050753e-03
-7.693491421116e-03
-3.010189146329e-03
-9.219058223799e-03
-5.197032408577e-03
 3.668576609002e-03
-9.323186306881e-03
 5.281597418884e-03
 3.650366613886e-03
 5.936029143949e-03
 9.205709303566e-03
 1.340013928676e-03
 5.800584373846e-03
-9.197365612438e-03
 1.334371957249e-03
 7.195777552226e-04
 5.995136778779e-03
-4.925301722832e-04
 3.047381654908e-03
-1.843928397297e-03
-5.058954422721e-03
 6.967908660325e-04
-6.015651022402e-03
-4.846506697423e-04
 3.045625684952e-03
 1.837392177345e-03
-5.062790155056e-03
-2.142153752482e-03
 3.296285100696e-03
 3.408015158748e-03
-2.142740472158e-03
-3.289264424246e-03
 3.414975462958e-03
 4.279780818377e-03
 4.122787098153e-03
-4.448367735211e-03
 8.778334402197e-04
 3.590712178973e-03
 4.591009863028e-03
 4.283619507701e-03
-4.134935226658e-03
-4.478371017836e-03
 8.981390916055e-04
-3.599689901978e-03
 4.595657786918e-03
