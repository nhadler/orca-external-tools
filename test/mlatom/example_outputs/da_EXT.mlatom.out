

  !---------------------------------------------------------------------------!
  !                                                                           !
  !     MLatom: a Package for Atomistic Simulations with Machine Learning     !
  !                             MLatom 3.xx.x                                 !
  !                                   @                                       !
  !                 Xiamen Atomistic Computing Suite (XACS)                   !
  !                                                                           !
  !                http://mlatom.com/ @ https://XACScloud.com                 !
  !                                                                           !
  !            MIT License (modified to request proper citations)             !
  !                   Copyright (c) 2013- Pavlo O. Dral                       !
  !                           http://dr-dral.com/                             !
  !                                                                           !
  ! Permission is hereby granted, free of charge, to any person obtaining a   !
  ! copy of this software and associated documentation files (the "Software"),!
  ! to deal in the Software without restriction, including without limitation !
  ! the rights to use, copy, modify, merge, publish, distribute, sublicense,  !
  ! and/or sell copies of the Software, and to permit persons to whom the     !
  ! Software is furnished to do so, subject to the following conditions:      !
  !                                                                           !
  ! The above copyright notice and this permission notice shall be included   !
  ! in all copies or substantial portions of the Software.                    !
  ! When this Software or its derivatives are used                            !
  ! in scientific publications, it shall be cited as:                         !
  !                                                                           !
  ! Pavlo O. Dral, Fuchun Ge, Yi-Fan Hou, Peikun Zheng, Yuxinxin Chen,        !
  ! Mario Barbatti, Olexandr Isayev, Cheng Wang, Bao-Xin Xue,                 !
  ! Max Pinheiro Jr, Yuming Su, Yiheng Dai, Yangtao Chen, Lina Zhang,         !
  ! Shuang Zhang, Arif Ullah, Quanhao Zhang, Yanchi Ou.                       !
  ! J. Chem. Theory Comput. 2024, 20, 1193-1213.                              !
  !                                                                           !
  ! Pavlo O. Dral, Fuchun Ge, Yi-Fan Hou, Peikun Zheng, Yuxinxin Chen,        !
  ! Bao-Xin Xue, Mikolaj Martyka, Max Pinheiro Jr, Yuming Su, Yiheng Dai,     !
  ! Yangtao Chen, Shuang Zhang, Lina Zhang, Arif Ullah, Quanhao Zhang,        !
  ! Sebastian V. Pios, Yanchi Ou, Matheus O. Bispo, Vignesh B. Kumar,         !
  ! Xin-Yu Tong,                                                              !
  ! MLatom: A Package for Atomistic Simulations with Machine Learning,        !
  ! version 3.xx.x, Xiamen University, Xiamen, China, 2013-2024.              !
  !                                                                           !
  ! The citations for MLatom's interfaces and features shall be eventually    !
  ! included too. See header.py, ref.json and http://mlatom.com.              !
  !                                                                           !
  ! THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS   !
  ! OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF                !
  ! MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN !
  ! NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,  !
  ! DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR     !
  ! OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE !
  ! USE OR OTHER DEALINGS IN THE SOFTWARE.                                    !
  !                                                                           !
  !---------------------------------------------------------------------------!


 ============================================================================== 
 MLatom started on 04.05.2025 at 15:53:38
        with the following options:
        /mlatom/software/mlatom_user/mlatom/shell_cmd.py useMLmodel MLmodelType=ANI MLmodelIn=da_energy_ani.npz XYZfile=/mlatom/home/dral/soft/orca_external_tools/test/da_EXT.xyz charges=0 multiplicities=1 nthreads=1 YestFile=/mlatom/home/dral/soft/orca_external_tools/test/da_EXT.mlatom.energy YgradXYZestFile=/mlatom/home/dral/soft/orca_external_tools/test/da_EXT.mlatom.gradient
 ============================================================================== 

 ****************************************************************************** 

 You are going to use feature(s) listed below. 
 Please cite corresponding work(s) in your paper:

 TorchANI program:
   X. Gao, F. Ramezanghorbani, O. Isayev, J. S. Smith, A. E. Roitberg,
   J. Chem. Inf. Model. 2020, 60, 3408

 ANI model:
   J. S. Smith, O. Isayev, A. E. Roitberg, Chem. Sci. 2017, 8, 3192

 ****************************************************************************** 
model loaded from da_energy_ani.npz
 ============================================================================== 
 Wall-clock time: 2.57 s (0.04 min, 0.00 hours)

 MLatom terminated on 04.05.2025 at 15:53:40
 ============================================================================== 
